features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/ResolveResourceCallback.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.resources
+
+internal interface ResolveResourceCallback {
+    fun onResolved(resourceId: String, byteArray: ByteArray)
```

Comment:
very nit: maybe `bytes`/`rawBytes` or simply `resource` instead of `byteArray` as argument name make more sense.